src/components/Accordion/accordionChangelogData.ts

```diff
@@ -14,7 +14,18 @@ export const changelogData: ChangelogData[] = [
     version: "Prerelease",
     type: "Update",
     affects: ["Accessibility", "Functionality"],
-    notes: ["Adds logic to close accordion when 'esc' key is pressed"],
+    notes: [
+      "Adds logic to close accordion when element within panel is focused and 'esc' key is pressed",
```

Comment:
This is duplicated below.

Reply:
You can keep both but what I mean is you only need one object with "Prerelease" but now there are two. Unless if this was done because you don't think this will get deployed by 5/23. If that's the case then it's okay to keep both.

Reply:
I see! Okay. Will combine.